tests/suites/0_stateless/09_fuse_engine/09_0001_remote_insert.sql

```diff
@@ -16,6 +16,7 @@ SELECT * FROM t2;
 DROP TABLE t1;
 DROP TABLE t2;
 
+set max_threads = 1;
```

Comment:
why set `max_thread` to 1 ?

Reply:
`transform_compact`  just works in one thread.